### PR TITLE
Added docs for auth0-ember-simple-auth

### PR DESCRIPTION
**Pull request by benschwarz**
- [x] Make auth0-ember-simple-auth/examples/simple use `auth0-variables.js`

### CODE REVIEW

**Comment by benschwarz** [comment]
Believe this is covered and ready to go — although not sure how to test/verify it before deployed.

**Comment by benschwarz** [comment]
@ntotten Look :ok: to you?

**Comment by benschwarz** [comment]
As per: auth0/auth0-ember-simple-auth/pull/4

File: articles/client-platforms/emberjs.md [hunk 1]
```diff
@@ -8,16 +8,93 @@ tags:
...
 </div>
 
-The EmberJS integration is a wrapper on top of [Auth0 Lock](https://github.com/auth0/lock). Right now, the integration consists in a `Mixin` that will help you handle the Login and keep the `currentUser` always in scope.
+**Otherwise, if you already have an existing application, please follow the steps below.**
+
+@@includes.callback@@
+
+### 1. Getting started
+
+`auth0-ember-simple-auth` is an add-on for [simple-auth](http://ember-simple-auth.com), which can be installed via [ember-cli](http://www.ember-cli.com).
```
Comment (mgonto) [inline articles/client-platforms/emberjs.md:28]: Add a step that shows how to install it with `ember-cli`

File: articles/client-platforms/emberjs.md [hunk 2]
```diff
@@ -8,16 +8,93 @@ tags:
...
+import ApplicationRouteMixin from 'simple-auth/mixins/application-route-mixin';
+
+export default Ember.Route.extend(ApplicationRouteMixin, {
+  actions: {
+    sessionRequiresAuthentication: function(){
+      // Check out the docs for all the options:
+      // https://auth0.com/docs/libraries/lock/customization
+
+      // These options will request a refresh token and launch lock.js in popup mode
+      var lockOptions = {authParams:{scope: 'openid offline_access'}};
```
Comment (mgonto) [inline articles/client-platforms/emberjs.md]: remove offline_access from here as well

File: articles/client-platforms/emberjs.md [hunk 3]
```diff
@@ -8,16 +8,93 @@ tags:
...
+
+      this.get('session').authenticate('simple-auth-authenticator:lock', lockOptions);
```
Comment (mgonto) [inline articles/client-platforms/emberjs.md:87]: Is this showing Lock? If so, can you please say that

Comment (mgonto) [inline articles/client-platforms/emberjs.md:87]: Also explain where you get the JWT, the profile, etc. Check https://github.com/auth0/docs/blob/master/articles/client-platforms/react.md#3-lets-implement-the-login

File: articles/client-platforms/emberjs.md [hunk 4]
```diff
@@ -8,16 +8,93 @@ tags:
...
+// app/routes/home.js
+
+import Ember from 'ember';
+import AuthenticatedRouteMixin from 'simple-auth/mixins/authenticated-route-mixin';
+
+export default Ember.Route.extend(AuthenticatedRouteMixin);
+```
+
+### 4. You're done!
+
```
Comment (mgonto) [inline articles/client-platforms/emberjs.md:156]: Explain also how to show the profile information and how you'd perform secured calls to your API.

File: articles/client-platforms/emberjs.md (hunk 1 above)
Comment (benschwarz) [inline articles/client-platforms/emberjs.md:28]: Done

**Comment by benschwarz** [comment]
@mgonto I think this is ready to go

File: articles/client-platforms/emberjs.md [hunk 5]
```diff
@@ -8,16 +8,156 @@ tags:
...
+{{/if}}
+```
+
+### 5. You're authenticated!
+
+When you authenticate, your application will receive session data from the popup window — Then, session data will be stored in localStorage with a key of `ember_simple_auth:session`. The session object is a JSON object that contains your user profile data, JWT token and access token.
+
+You can access this session information in ember templates by using `{{session.secure}}`. So for example, if you wanted to say Hi to the user and show the associated avatar:
+
+```handlebars
```
Comment (benschwarz) [inline articles/client-platforms/emberjs.md:131]: Not 100% sure that this is a formatter

**Comment by ntotten** [comment]
Is this ready? Can we merge?

**Comment by mgonto** [comment]
Looks good. We can merge. @ntotten is it worth extracting the snippets from this one before merging as well?

**Pull request merged**